ui/create_restore_page.go

```diff
@@ -323,64 +307,222 @@ func (pg *createRestore) mainContent(gtx layout.Context) layout.Dimensions {
+				layout.Flexed(1, func(gtx C) D {
+					return inset.Layout(gtx, func(gtx C) D {
+						return pg.inputsGroup(gtx, pg.seedList, 6, 3)
+					})
+				}),
+				layout.Flexed(1, func(gtx C) D {
+					return pg.inputsGroup(gtx, pg.seedList, 6, 4)
+				}),
+			)
 		}),
 		layout.Rigid(func(gtx C) D {
-			txt := pg.theme.H6("Enter your seed phrase in the correct order")
-			txt.Alignment = text.Middle
-			return pg.centralize(gtx, func(gtx C) D {
-				return txt.Layout(gtx)
-			})
+			return pg.errLabel.Layout(gtx)
 		}),
 		layout.Rigid(func(gtx C) D {
-			return layout.Inset{Top: values.MarginPadding10, Bottom: values.MarginPadding10}.Layout(gtx, func(gtx C) D {
-				return pg.centralize(gtx, func(gtx C) D {
-					return pg.errLabel.Layout(gtx)
-				})
-			})
+			return pg.resetSeedFields.Layout(gtx)
 		}),
-		layout.Flexed(1, func(gtx C) D {
-			return layout.Center.Layout(gtx, func(gtx C) D {
-				return layout.Flex{}.Layout(gtx,
-					layout.Rigid(func(gtx C) D {
-						gtx.Constraints.Max.X = gtx.Constraints.Max.X / 2
-						return pg.inputsGroup(gtx, pg.seedListLeft, 16, 0)
-					}),
-					layout.Rigid(func(gtx C) D {
-						return pg.inputsGroup(gtx, pg.seedListRight, 17, 16)
-					}),
-				)
+	)
+
+}
+
+func (pg *createRestore) createPasswordPhase(gtx layout.Context) layout.Dimensions {
+	phaseContents := []func(gtx C) D{
+		func(gtx C) D {
+			card := pg.theme.Card()
+			card.Color = pg.theme.Color.LightGray
+			msg := "This spending password is required to sign transactions. Make sure to use a strong password and keep it safe."
+			return card.Layout(gtx, func(gtx C) D {
+				gtx.Constraints.Min.X = gtx.Constraints.Max.X
+				return layout.UniformInset(values.MarginPadding16).Layout(gtx, func(gtx C) D {
+					return layout.Flex{}.Layout(gtx,
+						layout.Rigid(func(gtx C) D {
+							inset := layout.Inset{
+								Right: values.MarginPadding10,
+								Top:   values.MarginPadding3,
+							}
+							return inset.Layout(gtx, func(gtx C) D {
+								return pg.alertIcon.Layout(gtx)
+							})
+						}),
+						layout.Rigid(func(gtx C) D {
+							return pg.theme.Body1(msg).Layout(gtx)
+						}),
+					)
+				})
 			})
-		}),
-		layout.Rigid(func(gtx C) D {
-			return pg.centralize(gtx, func(gtx C) D {
-				return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+		},
+		func(gtx C) D {
+			return pg.spendingPassword.Layout(gtx)
+		},
+		func(gtx C) D {
+			return pg.passwordStrength.Layout(gtx)
+		},
+		func(gtx C) D {
+			return pg.matchSpendingPassword.Layout(gtx)
+		},
+	}
+
+	return (&layout.List{Axis: layout.Vertical}).Layout(gtx, len(phaseContents), func(gtx C, i int) D {
+		return layout.UniformInset(values.MarginPadding5).Layout(gtx, phaseContents[i])
+	})
+}
+
+func (pg *createRestore) renameWalletPhase(gtx layout.Context) layout.Dimensions {
+	return pg.walletName.Layout(gtx)
+}
+
+func (pg *createRestore) restorePageSections(gtx layout.Context, title string, phaseProgress string, body layout.Widget) layout.Dimensions {
```

Comment:
<img width="391" alt="image" src="https://user-images.githubusercontent.com/27733432/116502987-e5500500-a8ac-11eb-8047-7eb725163261.png">

<img width="323" alt="image" src="https://user-images.githubusercontent.com/27733432/116503058-14667680-a8ad-11eb-8527-83932d8f51db.png">

Too much padding between the section title and the body. 
You currently have an Inset of 10Dp and a UniformPadding of 16Dp.